e2e/content-services/share-file/share-file.e2e.ts

```diff
@@ -142,18 +144,11 @@ describe('Share file', () => {
             await BrowserActions.closeMenuAndDialogs();
         });
 
-        it('[C286578] Should disable today option in expiration day calendar', async () => {
```

Comment:
why are you removing this test?

Reply:
it's a copy-pasted one, the same test is above :)

Reply:
hehe 😄